lib/operate.ts

```diff
@@ -242,10 +242,11 @@ const getProcessInstanceCallHierarchy: Endpoint<Pick<ProcessInstance, 'processIn
 
 const callHierarchySchema = z.object({
 	processInstanceKey: z.string(),
+	processDefinitionKey: z.string(),
 	processDefinitionName: z.string(),
 });
 type CallHierarchy = z.infer<typeof callHierarchySchema>;
-const getProcessInstanceCallHierarchyResponseBodySchema = getCollectionResponseBodySchema(callHierarchySchema);
+const getProcessInstanceCallHierarchyResponseBodySchema = z.array(callHierarchySchema);
```

Comment:
@abremard  Is there any reason why this endpoint doesn't follow the standard collection response pattern? Should we raise the issue with the backend team?